### PR TITLE
Add CalConnect templates

### PR DESCRIPTION
**Pull request by manuelfuenmayor**
*As requested in https://github.com/metanorma/metanorma/issues/53*

Added *Guide*, *Specification*, *Amendment*, and *Technical Corrigendum* templates.

### CODE REVIEW

**Comment by manuelfuenmayor** [comment]
FYI, a rebase was applied and forced pushed.

File: amendment/README.adoc [hunk 1]
```diff
@@ -0,0 +1,66 @@
+= CalConnect Template: Amendment
+
+This repository contains the CalConnect CSD Amendment document template.
```
Comment (ronaldtse) [inline amendment/README.adoc]: Remove the "CSD" phrase.

File: amendment/README.adoc [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+image:https://travis-ci.com/travis-account/my-repo-name.svg?branch=master["Build Status", link="https://travis-ci.com/travis-account/my-repo-name"]
```
Comment (ronaldtse) [inline amendment/README.adoc:5]: Remove this line since we don't use Travis anymore.

File: amendment/document.adoc [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+- report
+- standard
+- guide
+- specification
+- amendment
+- technical corrigendum
+////
+
+
+include::sections/00-foreword.adoc[]
```
Comment (opoudjis) [inline amendment/document.adoc:66]: Remember: blank lines between all include macros. Applies throughout

**Review by opoudjis — CHANGES_REQUESTED** [review]

File: amendment/document.adoc (hunk 3 above)
Comment (manuelfuenmayor) [reply amendment/document.adoc:66]: Fixed, thanks!

File: amendment/sections/03-terms_defs.adoc [hunk 4]
```diff
@@ -0,0 +1,65 @@
+
+[[terms]]
+== Terms, definitions and abbreviations
+
+=== Terms and definitions
+
+[[calendar-spam]]
```
Comment (ronaldtse) [inline amendment/sections/03-terms_defs.adoc]: Remove all anchors from Terms section and use `term:[]` command.

**Review by ronaldtse — CHANGES_REQUESTED** [review]

File: amendment/README.adoc (hunk 1 above)
Comment (manuelfuenmayor) [reply amendment/README.adoc]: Fixed.

File: amendment/README.adoc (hunk 2 above)
Comment (manuelfuenmayor) [reply amendment/README.adoc:5]: Fixed.

File: amendment/sections/03-terms_defs.adoc (hunk 4 above)
Comment (manuelfuenmayor) [reply amendment/sections/03-terms_defs.adoc]: Fixed

**Comment by manuelfuenmayor** [comment]
Having applied the requested changes, I'm merging now.

**Pull request merged**